src/main/resources/application.properties

```diff
@@ -25,7 +25,7 @@ spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
 server.compression.enabled=false
 
 spring.data.mongodb.uri=${MONGODB_URI:${env.MONGODB_URI:mongodb+srv://fakeUsername:fakePassword@cluster0.ulqcw.mongodb.net/fakeDatabase?retryWrites=true&w=majority}}
-
+spring.data.mongodb.database=database
```

Comment:
This is already defined at the bottom of this file. Please remove one or the other.

Reply:
Oops thanks